components/TextFields/src/MDCTextInputControllerDefault.m

```diff
@@ -1153,14 +1154,14 @@ - (void)updateFontsForDynamicType {
  LTR.
 
  The vertical layout is, at most complex, this form:
- MDCTextInputDefaultVerticalPadding +                                        // Top padding
+ MDCTextInputDefaultPadding +                                         // Top padding
  MDCRint(self.textInput.placeholderLabel.font.lineHeight * scale) +   // Placeholder when up
- MDCTextInputDefaultVerticalHalfPadding +                                    // Small padding
- MDCRint(MAX(self.textInput.font.lineHeight,                          // Text field or placeholder
-              self.textInput.placeholderLabel.font.lineHeight)) +
- MDCTextInputDefaultVerticalHalfPadding +                                    // Small padding
- --Underline--                                                        // Underline (at active height)
- underlineLabelsOffset                                                      // Padding and labels
+ MDCTextInputDefaultVerticalHalfPadding +                             // Small padding
+ MDCCeil(MAX(self.textInput.font.lineHeight,                          // Text field or placeholder
```

Comment:
Why the switch from MDCRint to MDCCeil?

Reply:
This comment was out of date which is why I was changing it. I haven't used Rint for estimating actual line height in a while.